Jenkinsfile

```diff
@@ -67,6 +67,7 @@ pipeline {
                     stage('System Tests Serial') {
                         steps {
                             sh """
+                            touch log/${NODENAME}/system_test/serial_test_run
```

Comment:
If the parallel test of the system test fails, the serial tests will not be triggered, which may lead the user to think that everything is ok for a certain package, while it was not actually tested with the serial test. E.g. If the parallel test for cosmo fails but passes for int2lm, no serial test will be executed for both of the packages. Thus it looks like all tests passed for int2lm, while we actually don't know about the serial test, as it didn't run.

In case the above file has not been created, we know that Jenkins did not run any serial test within the system test. In that case a warning is being issued to remind us that not all tests have been run. 

If all parallel tests pass but not all serial tests, the serial tests which failed will be shown nevertheless. So there won't be any false positive.

Please correct me @jonasjucker

Reply:
Yes, that should actually not be the case...

Reply:
You're absolutely right.

Reply:
Thanks, so the warning should be implemented in your pull request?